docs/pages/resources/changelog.md

```diff
@@ -26,6 +26,7 @@ If you're a CDN user, you must update your path to point to `cdn/` instead of `d
 - Added a `cdn/` distribution for bundled dependencies (imports for npm users remain the same) [#1369](https://github.com/shoelace-style/shoelace/pull/1369)
 - Added the `checkbox` part and related exported parts to `<sl-tree-item>` so you can target it with CSS [#1318](https://github.com/shoelace-style/shoelace/discussions/1318)
 - Added the `submenu-icon` part to `<sl-menu-item>` (submenus have not been implemented yet, but this part is required to allow customizations)
+- Added the ability to use Sprite Sheets when using `<sl-icon>` via a custom resolver.
```

Comment:
Not sure if you want to mention moving the method, from the prototype to the instance.

As you know any observable behavior is an API for someone somewhere... :)

Reply:
Looks like `resolveIcon()` went from a private static to a private instance method. Even though it's technically not private in JS-land, we treat it as if it _isn't_ part of the public API so no need to add it here.